build-system/tasks/e2e/package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "private": true,
+  "name": "gulp-e2e",
+  "version": "0.1.0",
+  "description": "Gulp e2e",  
+  "dependencies": {
```

Comment:
Since we aren't exactly shipping this directory, should this be `devDependencies`? (Not sure if I'm right about this.)

Reply:
It creates its own `node_modules` folder in `build-system/tasks/e2e` so it might not matter. I assume we don't ship anything in `build-system`?